lib/vitalsigns/command.rb

```diff
@@ -2,15 +2,25 @@
 require 'json'
 
 module Vitalsigns
+  # Base class for commands
   class Command
-    def execute(command, options={})
-      Open3.popen3(command) do |stdin, stdout, stderr, wait_thr|
-        return stdout.read
+    def execute(command, _options = {})
```

Comment:
Rubocop raise the warning message for unused arguments.
To avoid these messages I have used _options instead of options. As In Ruby, usually we use _ for unused block parameters.